### ISSUE TITLE
[bug] All logs are being duplicated

### ISSUE THREAD

**Issue by nepython** [openwisp/openwisp-monitoring#131]
On the master branch of openwisp-monitoring, I found that logs are duplicated as can be seen below. I am observing this for the first time.

![Screenshot from 2020-06-15 18-30-20](https://user-images.githubusercontent.com/54471024/84661333-bedde100-af09-11ea-90bd-9ca3441e0734.png)

**Comment by PabloCastellano** [comment]
This change might be related https://github.com/openwisp/openwisp-monitoring/commit/a51467c1effc3f66ae63431ad2d498c485343129

Since we have a root logger now, we probably have to stop propagating from the other loggers:

```
    'loggers': {
        'django': {
            'handlers': ['console'],
            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
            'propagate': False,
        },
```

**Comment by nepython** [comment]
Closed by #137
